### PR TITLE
[examples] adding the references for the new video API

### PR DESCRIPTION
**Pull request by bjuncek**
Simple IPYNB reference for using the new video API.

### CODE REVIEW

File: examples/python/VideoAPI.ipynb
Comment (fmassa) [inline examples/python/VideoAPI.ipynb:338]: This size for audio looks wrong. I think you might have concatenated on the wrong dimension

File: examples/python/VideoAPI.ipynb
Comment (fmassa) [inline examples/python/VideoAPI.ipynb:527]: nit: update to the final path of `Video`

File: examples/python/VideoAPI.ipynb
Comment (fmassa) [inline examples/python/VideoAPI.ipynb:579]: This can now be simplified with just `t.Resize((112, 112))` now that we support tensor as input to video ops

File: examples/python/VideoAPI.ipynb
Comment (fmassa) [inline examples/python/VideoAPI.ipynb:1]: Can we change the name of the video to follow `snake_case`? None of the files in the repo (apart from from C++ files) use CamelCase

**Review by fmassa — COMMENTED** [review]
Thanks for the changes!

I think there are a few minor bugs, but once those are fixed we can merge this to unblock and move forward

File: examples/python/VideoAPI.ipynb
Comment (bjuncek) [reply examples/python/VideoAPI.ipynb:1]: do you mean the name of the file?
done if so

File: examples/python/README.md
Comment (fmassa) [inline examples/python/README.md:6]: Needs to update here as well with the new names
```suggestion
- [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/pytorch/vision/blob/master/examples/python/video_api.ipynb)
[Example of VideoAPI](https://github.com/pytorch/vision/blob/master/examples/python/video_api.ipynb)
```

File: examples/python/README.md
Comment (fmassa) [inline examples/python/README.md:18]: typo: furthermore, and I would frame this differently.

This whole file will need a rewrite as it was very specific to one example
```suggestion
Furthermore, previously we used to provide a very high-level API for video decoding which left little control to the user. We're now expanding that API (and replacing it in the future) with a lower-level API that allows the user a frame-based access to a video.
```

**Review by fmassa — APPROVED** [review]
Approving to move forward with this

**Pull request merged**
